### DISCUSSION TITLE
Maintainers Needed

### DISCUSSION THREAD

**Discussion by VoyTechnology** [terra-farm/terraform-provider-virtualbox#117] [Support]
Hi all!

With help of @ringods I have been the primary maintainer of this repository for a while. While I love doing it and will continue to do so, I would greatly appreciate any help. There is a tonne to do, mainly to bring this provider to the level of other providers which have emerged in the Terraform community.

No matter what your interests are there is lot of work ahead of us. If you are interested, just comment below, start reviewing some PRs and reviewing open issues. In the coming days I will be filling out the Kanban board and prioritising issues and new features.

**Comment by BalajiTechs** [comment]
@VoyTechnology  I can be helping hand if it works. Please let me know.

**Reply by VoyTechnology** [reply]
Hi @BalajiTechs, can you take a look at https://github.com/terra-farm/terraform-provider-virtualbox/pull/151?

**Comment by essemenoff** [comment]
I am happy to help you

**Reply by VoyTechnology** [reply]
Hi @essemenoff, do you mind taking a look at https://github.com/terra-farm/terraform-provider-virtualbox/pull/152?

**Comment by ivocalado** [comment]
@VoyTechnology , I can help on that

**Reply by VoyTechnology** [reply]
Hi @ivocalado, I have tagged you in https://github.com/terra-farm/go-virtualbox/pull/42.

**Comment by carlosjpr-collab** [comment]
i can help
